src/modules/createTasks.js

```diff
@@ -0,0 +1,91 @@
+const toDoList = document.querySelector('#todo-lists');
+
+export default class CreateTasks {
+    static createTask = () => {
+      const task = document.createElement('div');
+      task.className = 'todo-list-container';
+      toDoList.appendChild(task);
+      const checkbox = document.createElement('input');
+      checkbox.type = 'checkbox';
+      const content = document.createElement('p');
+      content.className = 'content';
+      const dots = document.createElement('i');
+      dots.className = 'dots fa-solid fa-ellipsis-vertical';
+      const trash = document.createElement('i');
+      trash.className = 'alt-fa fa-solid fa-trash';
+
+      task.append(checkbox, content, dots, trash);
+
+      checkbox.addEventListener('click', () => {
+        task.classList.toggle('checked-holder');
+        content.classList.toggle('line-through');
+        trash.classList.toggle('trash-active');
+        dots.classList.toggle('dots-remove');
+
+        const localData = JSON.parse(localStorage.getItem('todo'));
+        const filter = [];
+        const taskChecked = document.querySelectorAll('.todo-list-container');
+        for (let i = 0; i < localData.length; i += 1) {
+          if (taskChecked[i].classList.contains('checked-holder')) {
+            localData[i].completed = true;
```

Comment:
User when checking an item the `completed` value doe not become `true` in local storage. Kindly make sure that the value gets `true` on checked and `false` on unchecked.